app/(root)/case-studies/[id]/page.tsx

```diff
@@ -1,7 +1,11 @@
 import React from 'react';
+import { ProjectDetailsProps } from '@/types';
+import ProjectDetails from '@/components/project-details/ProjectDetails';
```

Comment:
Instead of using Page as the component name, use something more descriptive such as ProjectDeatils. 

- Please be sure to refer back to the previous notes for other changes to include here.

Reply:
Since this route deals with case studies the dynamic page is CaseStudiesDynamic, and the main page CaseStudy